android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStorageModule.java

```diff
@@ -46,37 +46,6 @@
   private ReactDatabaseSupplier mReactDatabaseSupplier;
   private boolean mShuttingDown = false;
 
-  // Adapted from https://android.googlesource.com/platform/frameworks/base.git/+/1488a3a19d4681a41fb45570c15e14d99db1cb66/core/java/android/os/AsyncTask.java#237
-  private class SerialExecutor implements Executor {
```

Comment:
Detox is using this implementation detail in its environment setup, so I had to make this class public and available in Next storage (even tho it's not used)

Reply:
Can you add this comment in the code so it doesn't get lost?